### ISSUE TITLE
Problem: SumOfArray

### ISSUE THREAD

**Issue by SiddharthChaberia** [WoMakersCode/challenges-algorithms#66]
```
def simpleArraySum(a):
    return sum(a)
```
Hopefully this counts as a contribution for Hacktober2022 ✨

**Comment by Dhanush-2605** [comment]
hi, @SiddharthChaberia  can I work on this?

**Comment by SiddharthChaberia** [comment]
Sure i guess? 😅

**Comment by Dhanush-2605** [comment]
hi @SiddharthChaberia  can you please check it

**Comment by Dhanush-2605** [comment]
hi, @SiddharthChaberia  can you please verify my code and let me know if I want to make any changes

**Comment by SiddharthChaberia** [comment]
sorry, but i m not much familiar with github.. where can i see ur approach?

**Comment by Dhanush-2605** [comment]
@SiddharthChaberia  it is in pull request section

**Comment by Dhanush-2605** [comment]
![image](https://user-images.githubusercontent.com/90703523/193587825-7f3e9b72-f13a-4f44-961f-8db726118888.png)
this is my code

**Comment by SiddharthChaberia** [comment]
yeah, this is also correct, i just used inbuilt function to reduce the number of lines.. but ur ones is also good as it explains the inner part

**Comment by Dhanush-2605** [comment]
yeah, ok can you merge this PR?
